### PR TITLE
Introduce CI

### PR DESCRIPTION
**Pull request by hannesa2**
My plan it to introduce in each submodule CI first, and then in https://github.com/gitx/gitx

This is a pre-task to switch to target `10.9`

### CODE REVIEW

**Comment by hannesa2** [comment]
Here is builds https://github.com/hannesa2/MGScopeBar/pull/1 unfortunately in upstream not.
When I would be able to push to upstream, it would build here as well with first commit. So we have to wait, till it's merged

@jspiro Do you have access to `settings` here ? 
@tiennou Here I'm not a maintainer. Anyway, I hope someone will see this pull request

Any improvements are welcome ! I just begin to work with Xcode

**Comment by hannesa2** [comment]
Puhh, everyone make a new fork of a fork of a fork ...

image


Do we need the changes of https://github.com/chbeer/MGScopeBar.git as well ?

**Comment by tiennou** [comment]
I've fixed up the permission at the org-level, so you should have access now.

File: .github/workflows/BuildPR.yml
Comment (jspiro) [inline .github/workflows/BuildPR.yml:6]:
```suggestion
  build-mgscopebar:
```
Follows the github naming convention, since this will also show up in the branch protection and checks.

File: .github/workflows/BuildPR.yml
Comment (jspiro) [inline .github/workflows/BuildPR.yml:1]:
```suggestion
name: build-mgscopebar
```

File: .github/workflows/BuildPR.yml
Comment (jspiro) [inline .github/workflows/BuildPR.yml:7]:
```suggestion
    name: Build MGScopeBar
```

**Review by jspiro — CHANGES_REQUESTED** [review]
That's shockingly straightforward! Once this works for PRs, should we make a workflow for this too when we prepare a release, or have this also do a build upon merge (where are the artifacts stored?)?

**Comment by jspiro** [comment]
I've enabled branch protection – after merge, please enable the requirement for this check to succeed: https://github.com/gitx/MGScopeBar/settings/branch_protection_rules/19113317

**Comment by hannesa2** [comment]
> should we make a workflow for this too when we prepare a release, or have this also do a build upon merge (where are the artifacts stored?)?

I would make a workflow for a release (if required) but first I want to have the pull request flow done

**Comment by jspiro** [comment]
> > should we make a workflow for this too when we prepare a release, or have this also do a build upon merge (where are the artifacts stored?)?
> 
> I would make a workflow for a release (if required) but first I want to have the pull request flow 

Of course, of course. After!

File: .github/workflows/BuildPR.yml
Comment (hannesa2) [reply .github/workflows/BuildPR.yml:1]: With it, copy&paste needs some more work in the future 😉

File: .github/workflows/BuildPR.yml
Comment (hannesa2) [inline .github/workflows/BuildPR.yml:25]: Is `archive` or is `build` better ?

**Comment by hannesa2** [comment]
Ok, I applied renames

File: .github/workflows/BuildPR.yml
Comment (jspiro) [reply .github/workflows/BuildPR.yml:25]: Archive seems like a good idea, as it packages it up?

**Review by jspiro — APPROVED** [review]

File: .github/workflows/BuildPR.yml
Comment (hannesa2) [reply .github/workflows/BuildPR.yml:25]: I didn't check. Are there tests as well to run ?
Sorry, I don't now how to find it in Xcode. Maybe you tell it me

**Comment by hannesa2** [comment]
Ahhh, sorry, there is something happen with `origin` and `upstream` 
Rename commit is not on master

image

I will fix it

**Comment by hannesa2** [comment]
https://github.com/gitx/MGScopeBar/pull/new/RenameAction

**Pull request merged**
